doc/content/examples/example5a.md

```diff
@@ -0,0 +1,121 @@
+
+## Modeling in MASTODON
+
+!listing examples/ex05a/StickModel_accel_base.i
+
+The mesh is generated by importing an existing mesh file. The details of the types of supported mesh files can be found at [Getting Started](manuals/user/index.md). The stick-models and the foundation beams are modeled using 1D line elements and the foundation is modeled using 3D solid elements as shown in [fig:ex05input]. The displacements and rotations in X, Y and Z directions are defined in the variables section and the velocity, acceleration, rotational velocity and rotational acceleration are defined as the auxiliary variables.
+
+!media media/examples/ex05input.png
+       style=width:50%;margin-left:150px;float:center;
+       id=fig:ex05input
+       caption=Input model in MASTODON.
+
+
+ The stick models are created using the [LineElementMaster](syntax/Modules/TensorMechanics/index.md) action. [DynamicTensorMechanics](syntax/index.md) and [InertialForce](syntax/index.md) kernels are used to model the dynamics of the 3D solid elements. The [Newmark time integration](manuals/theory/index.md) parameters used in this problem correspond to the Newmark's average acceleration method, i.e. `beta = 0.25` and `gamma = 0.5`. A constant [Rayleigh damping](manuals/theory/index.md) of 5% is implemented using the values of `zeta = 0.00054388` and `eta = 3.266`. The section properties and the lumped masses described in [fig:stickmodel] are assigned to the different blocks of the model through the LineElementMaster action.
+
+The [ComputeElasticityBeam](syntax/index.md) block is used to create the elasticity tensor of the beam elements, using the parameters, Young's Modulus, Poisson's ratio, and shear coefficient. The forces and moments in the beam are calculated in the [ComputeBeamResultant](syntax/index.md) block.
+
+The nodes at the connection between the beam elements and the solid elements at the foundation are restrained in local rotation. The acceleration time series shown in [fig:5a_accelx], [fig:5a_accely] and [fig:5a_accelz]  is applied at the base of the foundation in X, Y and Z directions respectively using the BCs block. The ground motion accelerations are extracted from the .csv files and a scale factor of 32.2 is used to convert the units of acceleration from 'g' to 'ft/${s}^2$'.
+
+!row!
+
+!col! small=12 medium=6 large=6
+!media media/examples/5a_accelx.png
+       style=width:400px id=fig:5a_accelx
+       caption=Input acceleration time series in X-direction
+!col-end!
+
+!col! small=12 medium=6 large=6
+!media media/examples/5a_accely.png
+       style=width:400px id=fig:5a_accely
+       caption=Input acceleration time series in Y-direction
+!col-end!
+
+!row-end!
+
+!row!
+
+!col! small=12 medium=6 large=6
+!media media/examples/5a_accelz.png
+       style=width:400px id=fig:5a_accelz
+       caption=Input acceleration time series in Z-direction
+!col-end!
+
+!row-end!
+
```

Comment:
Please add figure 7 with the response spectra of the input. Response spectra are more important than the acceleration histories here.